src/core/solvers/ddp.cpp

```diff
@@ -258,7 +258,8 @@ void SolverDDP::backwardPass() {
       }
       Vxx_[t].noalias() -= Qxu_[t].leftCols(nu) * K_[t].topRows(nu);
     }
-    Vxx_[t] = 0.5 * (Vxx_[t] + Vxx_[t].transpose()).eval();
+    Vxx_tmp_ = Vxx_[t] + Vxx_[t].transpose();
+    Vxx_[t].noalias() = 0.5 * Vxx_tmp_;
```

Comment:
Putting 0.5 in the above line will conduct to even better performances I imagine.
Another way to improve this would be to do the operation in two passes. But I do agree, improving timings takes times.

Reply:
See also my comment in https://github.com/loco-3d/crocoddyl/pull/937#commitcomment-50062484

Reply:
Thank you, amended :-)